### PR TITLE
feat(website/patterns status): redesign more readable pages for patte…

### PR DESCRIPTION
**Pull request by gael-boyenval**
…rn status

## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a breaking change?

read [What is a breaking change ?](https://mozaic.adeo.cloud/Contributing/Prerequisite/GitConventions/#breaking-changes-)

- [ ] Yes
- [x] No

## Describe the changes

Github issue number or Jira issue URL: N/A

redesign pattern status branch

### CODE REVIEW

File: packages/gatsby-theme-styleguide/src/gatsby-components/StatusFlag/__tests__/StatusFlag.test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import Enzyme from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
```
Comment (mohamedMok) [inline packages/gatsby-theme-styleguide/src/gatsby-components/StatusFlag/__tests__/StatusFlag.test.js]: Same here, no Enzyme or Adapter

**Comment by nherwegh** [comment]
@gael-boyenval I don't know where it is located in the files but the `Pattern Status` link in homepage menu doesn't work

File: packages/gatsby-theme-styleguide/src/gatsby-components/PatternsStatus/PatternsStatus.js [hunk 2]
```diff
@@ -1,74 +1,238 @@
...
+        <PatternsStatus data={data.Components} />
+        <h3>Statuses meaning</h3>
+        <p>
+          <StatusFlag status="wip" />
+          <b>&nbsp;&nbsp;Work In Progress</b> : The pattern may have code and
+          documentation, but it should no be used.
+        </p>
+        <p>
+          <StatusFlag status="bêta" />
+          <b>&nbsp;&nbsp;Bêta</b> : The pattern is documented, and accessible in
```
Comment (nherwegh) [inline packages/gatsby-theme-styleguide/src/gatsby-components/PatternsStatus/PatternsStatus.js]: no ",'

File: packages/gatsby-theme-styleguide/src/gatsby-components/PatternsStatus/PatternsStatus.js [hunk 3]
```diff
@@ -1,74 +1,238 @@
...
+          the specified platform, but it may have breaking changes before being
```
Comment (nherwegh) [inline packages/gatsby-theme-styleguide/src/gatsby-components/PatternsStatus/PatternsStatus.js]: "'," should be after but

File: packages/gatsby-theme-styleguide/src/gatsby-components/PatternsStatus/PatternsStatus.js [hunk 4]
```diff
@@ -1,74 +1,238 @@
...
+      <div>
+        <h2>Foundations</h2>
+        <PatternsStatus data={data.Foundations} />
+        <h2>Components</h2>
+        <PatternsStatus data={data.Components} />
+        <h3>Statuses meaning</h3>
+        <p>
+          <StatusFlag status="wip" />
+          <b>&nbsp;&nbsp;Work In Progress</b> : The pattern may have code and
+          documentation, but it should no be used.
```
Comment (nherwegh) [inline packages/gatsby-theme-styleguide/src/gatsby-components/PatternsStatus/PatternsStatus.js]: "'," should be after but

File: packages/gatsby-theme-styleguide/src/gatsby-components/PatternsStatus/PatternsStatus.js [hunk 5]
```diff
@@ -1,74 +1,238 @@
...
+        </p>
+        <p>
+          <StatusFlag status="bêta" />
+          <b>&nbsp;&nbsp;Bêta</b> : The pattern is documented, and accessible in
+          the specified platform, but it may have breaking changes before being
+          labeled as stable. Basicaly, a bêta version of a pattern is waiting
+          for users (you) to test it and confirm us that it fit the teams needs.
```
Comment (nherwegh) [inline packages/gatsby-theme-styleguide/src/gatsby-components/PatternsStatus/PatternsStatus.js]: (…) confirm that it fits the needs of your team.

File: packages/gatsby-theme-styleguide/src/gatsby-components/PatternsStatus/PatternsStatus.js [hunk 6]
```diff
@@ -1,74 +1,238 @@
...
+          You should avoid using it in production, but if you do, be extra
+          careful when updating Mozaic.
+        </p>
+        <p>
+          <StatusFlag status="stable" />
+          <b>&nbsp;&nbsp;Stable</b> : The pattern is usable in production. The
+          mozaic teams can add features or fix bugs to it, as long there is no
```
Comment (nherwegh) [inline packages/gatsby-theme-styleguide/src/gatsby-components/PatternsStatus/PatternsStatus.js]: Mozaic's team & and also no ",'

**Comment by tiagoux** [comment]
> @gael-boyenval I don't know where it is located in the files but the `Pattern Status` link in homepage menu doesn't work

image

**Pull request merged**